### DISCUSSION TITLE
Import via .ssh/config file or other apps

### DISCUSSION THREAD

**Discussion by Julrouxxx** [isontheline/pro.webssh.net#750] [Ideas]
Hello,

It would be awesome if we could import all the SSH servers we had previously via the .ssh/config file. Also, why not importing from other apps (Termius for example). With a duplicate detection it will be even more perfect.

Thank you for this perfect software, much better and not subscription-based! (unlike one I cited here)

**Comment by isontheline** [comment]
@Julrouxxx Bonjour Julien 👋

Thank you so much for your feedback and support about WebSSH 🙏

Import feature has been planned a long time ago here : https://github.com/isontheline/pro.webssh.net/issues/252
I'm very late on it 😢

**Reply by Julrouxxx** [reply]
Oh I didn't notice it sorry

**Reply by isontheline** [reply]
No problem at all, I like your feedback
